quick_start/my_text_classifier_bert.jsonnet

```diff
@@ -0,0 +1,48 @@
+local bert_model = "bert-base-uncased";
+
+{
+    "dataset_reader" : {
+        "type": "classification-tsv",
+        "tokenizer": {
+            "word_splitter": "bert-basic"
+        },
+        "token_indexers": {
+            "bert": {
+                "type": "bert-pretrained",
+                "pretrained_model": bert_model,
+            }
+        }
+    },
+    "train_data_path": "data/movie_review/train.tsv",
+    "validation_data_path": "data/movie_review/dev.tsv",
+    "model": {
+        "type": "simple_classifier",
+        "embedder": {
+            "allow_unmatched_keys": true,
+            "embedder_to_indexer_map": {
+                "bert": ["bert", "bert-offsets"]
+            },
+            "token_embedders": {
+                "bert": {
+                    "type": "bert-pretrained",
+                    "pretrained_model": bert_model,
+                    "top_layer_only": true,
+                    "requires_grad": false
+                }
+            }
+        },
+        "encoder": {
+            "type": "bert_pooler",
```

Comment:
This *should* work with the other `pretrained-transformer` stuff that I recommended, though I haven't explicitly tested that.